samples/Uno.Toolkit.WinUI.Samples/Uno.Toolkit.WinUI.Samples.Mobile/iOS/Info.plist

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+	<key>CFBundleDisplayName</key>
+	<string>Uno.Toolkit.WinUI.Samples</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.companyname.Uno.Toolkit.WinUI.Samples</string>
```

Comment:
I'm wondering if the Bundle ID should be the same as the WinUI Xamarin.iOS one (uno.platform.toolkit). Similar to how it looks to be done for Gallery

Xamarin:
https://github.com/unoplatform/Uno.Gallery/blob/afb97a5f511ef703d5676382e841f9295a3ba9ba/Uno.Gallery/Uno.Gallery.iOS/Info.plist#L8

net6.0:
https://github.com/unoplatform/Uno.Gallery/blob/afb97a5f511ef703d5676382e841f9295a3ba9ba/Uno.Gallery/Uno.Gallery.Mobile/iOS/Info.plist#L8

Not sure if this really matters though

Reply:
Thanks